src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
+
+type Props = {
+  comments: Comment[],
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="CommentList">
+    {comments.map(comment => (
+      <li><CommentInfo key={comment.id} comment={comment} /></li>
```

Comment:
One tag per line.
Also, key must be on the top level of list items.
```suggestion
      <li key={comment.id}>
        <CommentInfo  comment={comment} />
      </li>
```